packages/status-js/src/client/community/community.ts

```diff
@@ -21,13 +23,15 @@ export class Community {
   private symmetricKey!: Uint8Array
   public description!: CommunityDescription
   public chats: Map<string, Chat>
+  #members: Map<string, Member>
```

Comment:
We can use [private class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields) properties for the internal state and expose it via getters.

Reply:
Like it, thanks for the intro!